.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: node_js
 sudo: true
-node_js:
-- stable
+node_js: stable
 script:
+- npm run lint
 - npm test
```

Comment:
I would prefer it if the command `npm test` ran both lint and the tests - otherwise I'm not going to remember to lint before pushing code, and I wouldn't expect other people to know to run a non-standard command either.